### ISSUE TITLE
Ensure cross network payment message upgradeability

### ISSUE THREAD

**Issue by goranjovic** [status-im/status-react#6493]
### User Story

As a user I want my payment messages to work gracefully between different versions of app using different networks.
As a developer I want to be able to add/remove selection of supported networks easily

### Description

*Type*: Feature

*Summary*: Receiving messages from future versions of the app should work reasonably well and without broken behavior and/or crashes, because future version sent a payment or request on different networks not known by the past version.

This task involves creation of a contrived PR build which would represent a hypothetical future version with some networks added and some removed. That build would then communicate with "this" build that represents the old version.

Following cases can occur:

1. Future sends/requests native currency of network N to/from Past and network N is  not known by Past. 

In this case, Past should receive and display the message (even partially, possibly omitting e.g. fiat value) but indicate that network and its currency are not known. 

2. Past version sends/requests on network M from Future version and network M was removed from Future.

In this case, Future should display the message gracefully and indicate that the network was removed from the selection. It should be distinguished from the case when network is not yet added.

Both cases should work with both ERC-20 tokens and native currencies.
 
If any changes are required in order to pass these criteria, they should be made in a separate PR and merged, while discarding the "hypothetical future PR".

The original problem behind this issue was in PR [#6388](https://github.com/status-im/status-react/pull/6388#issuecomment-432549313) - point 2

This issue is equivalent to #6492 which deals with the same thing with tokens.
Depends on #6491

**Comment by ghost** [comment]
This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.
